--- a/Model/ResourceModel/Address/Attribute/Source/Country.php	2018-11-19 12:01:14.000000000 +0100
+++ b/Model/ResourceModel/Address/Attribute/Source/Country.php	2019-02-24 01:39:25.223915069 +0100
@@ -49,7 +49,7 @@
      *
      * @return array
      */
-    public function getAllOptions()
+    public function getAllOptions($withEmpty = true, $defaultValues = false)
     {
         if (!$this->_options) {
             $this->_options = $this->_createCountriesCollection()->loadByStore(
